pelicanconf.py

```diff
@@ -7,6 +7,10 @@
 
 PATH = 'content'
 
+PAGE_PATHS = ['']
```

Comment:
Does this not control which directly to scan for pages? I think it should be kept as the default since we keep pages in the `pages/` directory in the repo.

Reply:
I think you're right. Didn't seam to break or change much but I reverted it.